src/constants/constants.js

```diff
@@ -22,7 +22,7 @@ export const AVAILABLE_COLORS_DARKER_SHADES = [
 export const DEFAULT_MINIMUM_ROTATION_ANGLE = -10;
 export const DEFAULT_MAXIMUM_ROTATION_ANGLE = 10;
 
-export const RE_FETCH_INTERVAL = 1500; // Default: 1500
+export const SMALL_DELAY_REFOCUS_MS = 20;
```

Comment:
It might not be highlighted in this PR, but I think I've never seen this variable being use, does it?

Reply:
Yes, this is a very short delay I use before resetting the focus on the note being edited. It leaves some time to the browser to define the element to be focused before changing it.

It is used here: https://github.com/graasp/graasp-app-sticky-notes/blob/8e7f4bcddf9da608a0f4ea73c51b248c2f78162b/src/components/main/note/edit-view/EditViewTitle.js#L49